README.md

```diff
@@ -1,12 +1,40 @@
 # Missed Issues
 
-A CLI tool for finding tickets that mention a team by no one has responded to from the team.
+A CLI tool for finding issues that mention a team and no one from the team has responded to the issue.
 
-`missed-issues --org mapbox --team teamname`
+`missed-issues --org mapbox --team teamname --token github_token`
 
 This will list all issues in the mapbox org that mention `@mapbox/teamname` where no member of that team has replied to the issue since the mention.
 
 ## Flags
 
-- `--max-issues`: the max number of issues to return in the search result before filtering. Defaults to 100.
+**Required**
 
+- `--org`: the name of the org you want to search.
+- `--team`: the name of the team you are checking into. This MUST not contain the name of the org.
+- `--token`: a GitHub token with `read:org, repo` access
+
+**Optional**
+
+- `--from`: the start date of your search range. All issues returned will have been edited on or after this date. Defaults to one week ago.
+- `--ignore-repos` a comma-separated list of repos that you wish to exclude from your search results. These are excluding via filtering and will thus still affect your max-issues number.
```

Comment:
make it clear that these need to be comma-separated with no spaces, or in quotes